script/validators/stat_validator.py

```diff
@@ -112,20 +124,23 @@ def parse(file_path):
 EXIT_FAILURE = -1
 
 if __name__ == '__main__':
-        # Launch Peloton with Copy Option
-        peloton = subprocess.Popen(['exec ' + PELOTON_BIN + ' -port 54321 -stats_mode 1'], shell=True)
-
-        # start jdbc test
-        os.chdir(PELOTON_JDBC_SCRIPT_DIR)
-        jdbc = subprocess.Popen(['/bin/bash test_jdbc.sh copy ' + PELOTON_STAT_CSV], shell=True, stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT)
-        jdbc.stdout.readlines()
-
-        # kill peloton
-        peloton.kill()
-
-        # try parse the stat result
-        parse(PELOTON_STAT_CSV)
-        print("Parse successful")
-        sys.exit(EXIT_SUCCESS)
-
+    # Launch Peloton with Copy Option
+    PELOTON = subprocess.Popen(
+        ['exec ' + PELOTON_BIN + ' -port 54321 -stats_mode 1'], shell=True
+    )
+
+    # start jdbc test
+    os.chdir(PELOTON_JDBC_SCRIPT_DIR)
+    JDBC = subprocess.Popen(
+        ['/bin/bash test_jdbc.sh copy ' + PELOTON_STAT_CSV],
+        shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+    )
+    JDBC.stdout.readlines()
```

Comment:
What's the purpose of this line?

Reply:
I don't see any purpose of this line. Remove it and see if the behavior changes.

Reply:
But how do I test this script?